src/hooks/listHooks.js

```diff
@@ -64,6 +65,31 @@ export const useRenderItem = (query, navigation, options = {}) => {
       );
       break;
     }
+    case LIST_TYPES.TEXT_WITH_BIG_CARD_LIST: {
+      renderItem = ({ item, index }) => {
+        if (index === 0) {
+          return (
+            <BigCardListItem navigation={navigation} horizontal={options.horizontal} item={item} />
+          );
+        } else {
+          return (
+            <TextListItem
+              item={{
+                ...item,
+                bottomDivider: true
+              }}
+              {...{
+                navigation,
+                noSubtitle: options.noSubtitle,
+                rightImage: true,
```

Comment:
why is the image on the right here and for text list on the left? do we want that difference in the master branch? its both a text list with an image, right? i see no reason to have an image once on the left and once on the right.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/74e7eb7f2b529145ccbb1f97a902aa1981ad5341